server/src/main/java/org/elasticsearch/indices/recovery/RecoverySettings.java

```diff
@@ -90,24 +101,29 @@
 
     private volatile ByteSizeValue maxBytesPerSec;
     private volatile int maxConcurrentFileChunks;
+    private volatile int maxConcurrentOperations;
     private volatile SimpleRateLimiter rateLimiter;
     private volatile TimeValue retryDelayStateSync;
     private volatile TimeValue retryDelayNetwork;
     private volatile TimeValue activityTimeout;
     private volatile TimeValue internalActionTimeout;
+    private volatile TimeValue internalActionRetryTimeout;
     private volatile TimeValue internalActionLongTimeout;
 
     private volatile ByteSizeValue chunkSize = DEFAULT_CHUNK_SIZE;
 
     public RecoverySettings(Settings settings, ClusterSettings clusterSettings) {
 
         this.retryDelayStateSync = INDICES_RECOVERY_RETRY_DELAY_STATE_SYNC_SETTING.get(settings);
+        this.maxConcurrentFileChunks = INDICES_RECOVERY_MAX_CONCURRENT_FILE_CHUNKS_SETTING.get(settings);
```

Comment:
Ah this is a conflict resolution mistake by me, the commit was already backported, see https://github.com/crate/crate/pull/11805/files/08c1cc149514157c7216ffb5d8c3344264feeb29..0316d453e6bffbccf5fe9dd9f851e0fa093e8554#diff-9e0f47bc6c623b00660868f85caccbbba17dd27c9b22113157b614330ed8641bR122. Will fix.

Reply:
👍